runhouse/servers/cluster_servlet.py

```diff
@@ -258,46 +271,37 @@ async def aperiodic_cluster_checks(self):
                     break
 
                 logger.debug("Performing cluster checks")
-                status: ResourceStatusData = await self.astatus()
+                status, den_resp = await self.astatus(
+                    send_to_den=should_send_status_and_logs_to_den
+                )
+
                 if should_update_autostop:
                     logger.debug("Updating autostop")
                     await self._update_autostop(status)
 
                 if not should_send_status_and_logs_to_den:
```

Comment:
if this is `False` why do we even call `astatus` above?

Reply:
or is it bc we need some of that status data to update the autostop?

Reply:
Because `self._update_autostop` uses the status info. The `_update_autostop` function checks if there are running functions at the moment, and updates the autostop accordingly.